docs/technical_reference/property_models/ADM1.rst

```diff
@@ -178,6 +182,26 @@ Process Rate Equations
    "Decay of X_ac", ":math:`\rho_{18} = k_{dec, X_{ac}} C_{X_{ac}}`"
    "Decay of X_h2", ":math:`\rho_{19} = k_{dec, X_{h2}} C_{X_{h2}}`"
 
+Acid-Base Equilibrium Constraints
```

Comment:
Not all of these are acid-base equilib constraints. Maybe we can stay generic for now and just write something like "Additional Constraints"